src/components/selectable/selectable_templates/selectable_template_sitewide.tsx

```diff
@@ -153,6 +153,15 @@ export const EuiSelectableTemplateSitewide: FunctionComponent<EuiSelectableTempl
     setPopoverIsOpen(true);
   };
 
+  const onSearchKeydown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    searchProps && searchProps.onKeyDown && searchProps.onKeyDown(e);
```

Comment:
Syntax simplification/suggestion
```suggestion
    searchProp?.onKeyDown?.(e);
```

EDIT: I see that the other `on` events in the file are using older `&&` syntax instead of optional chaining - feel free to change those instances as well, up to you :)

Reply:
Fixed this commit.
https://github.com/elastic/eui/pull/5886/commits/896d716b26c513a031ebd4382fa861e14fb9f51e